### PR TITLE
Vox Announcements for 2 Oct 2019

### PR DESCRIPTION
**Pull request by N3X15**
I noticed TG made some improvements and additions a while ago, so I integrated them and made additional changes and improvements.  I can't get a hold of David for their pronunciation fixes, though.

We're up to _over 900_ new words, and I've recompressed all of them using TG's technique so everything's smaller. In addition, I've added some sfx (_ prefix), and changed some announcements. The [ss13-vox](https://github.com/N3X15/ss13-vox) repo has all of the raw files and the generator itself if you'd like to see the text representations. 

In the future, **please submit a PR there for future word, pronunciation, or tooling changes**.  There are a ton of servers that use ss13-vox, and they can all benefit.

## Notes
* The new data/vox_data.json file is there for users who use external tools.  Apparently, this is a thing. A JSON schema coming eventually when I finalize the structure and requirements.
* Some words have pronunciation problems. I am slowly fixing these, but it's not simple due to how Festival TTS works.
* Sounds like shit due to recompression.  This is deliberate, and reduces file size by 80%.

The current recompression settings (supplied to ffmpeg): 
```python
RECOMPRESS_ARGS = ['-c:a', 'libvorbis', '-ac', '1', '-ar', '16000', '-q:a', '0', '-speed', '0', '-threads', '8', '-y']
```

In short, this means:
* libvorbis codec
* Forced to single channel (mono)
* 16kHz sampling rate
* worst audio quality possible (1/10)

[qol][sound]

## Changelog
:cl:
 * soundadd: VOX Announcements Update! 900+ new words, changed announcements, and other improvements.

### CODE REVIEW

**Comment by ArchSunder** [comment]
cock and ball torture

**Comment by boy2mantwicethefam** [comment]
>deleted a comment

bruh
Anyway uhhhh I hope it's the vox announcer and not the gay female announcer tbh

**Comment by SysadminJeroen** [comment]
You compressed the audio so far down I can easily hear the artifacts.

**Comment by N3X15** [comment]
I can tweak the FFMPEG settings fairly easily.  There's a tradeoff between quality and size, though, and TG's settings removed around 80% of the size on disk.

**Comment by SonixApache** [comment]
madman

**Comment by press-mark** [comment]
would it be possible to log words that AI's try to say but aren't in the sounds, that way you could check it once a month or so and add the most common words that fail

**Comment by N3X15** [comment]
Too much effort for a system I wouldn't have access to.  Easier just to have people make their own wishlists.

**Comment by FudgePucker** [comment]
Woop woop, captain behind has been breached, woop woop.

**Comment by Kharkov72** [comment]
>Fixed six sounding like dicks. Only took me two years.

no...

**Comment by N3X15** [comment]
OK, I think I'm going to call this batch of updates good for now, unless other changes are needed.  I look forward to player feedback; I expect the sampling rate and quality may need a nudge up later, but the RSC download size may be a factor against that, given how large the download is already.

**Comment by SonixApache** [comment]
hey since you're doing this thing again can i ask you to make a better announcement for the connection lost event or should i do it elsewhere

**Comment by Probe1** [comment]
I'll testmerge and do a spotcheck then merge it to production today.  Just as soon as pomf gets up and I find out why the test servers down.

**Comment by N3X15** [comment]
@Probe1 Is it OK to remove the WiP label, or did you need something specific?

**Comment by N3X15** [comment]
> 
> 
> hey since you're doing this thing again can i ask you to make a better announcement for the connection lost event or should i do it elsewhere

Right now I'm in the middle of rewriting how voices are done in the generation script so that they're more modular (coming soon, [masculine voices](https://www.nexisonline.net/downloads/vox_login-rms-processed.ogg) genned at the same time as feminine voices), so not right now.  

What I'd like to try instead is sort of a batch system so we can have an issue with all the VOX phrase requests, and then we can discuss them and then generate a new batch every [convenient timeframe], which makes reviews less of a cluster.

**Comment by abernd** [comment]
In the past I mined the logs to generate a list of top words said in character over the server lifetime. If you want to batch add words, it might be helpful. Here's a list of the top 10,000 words by frequency. 

https://pastebin.com/Nj06qXfy

**Comment by N3X15** [comment]
Thanks, I'll look at adding that in the next batch PR.

**Comment by Probe1** [comment]
Sorry for sitting on this for so long.  I really wanted to test it out first but the test servers been a shit like we discussed in the thread.

**Comment by FudgePucker** [comment]
Irrelevant to this PR, but it would be really funny if you could do VOX announcements from a communications computer with high enough access.
I want to use VOX to shitpost but dont want to have to play AI

**Comment by Probe1** [comment]
VOX announcements are one of the toys AI get.  Stealing other kids toys is bad.
